features/calculation_weekly_frequency.feature

```diff
@@ -0,0 +1,26 @@
+Feature:
+  As a enrolee into a Workplace Pension,
+  I would like to calculate pension contribution on my weekly salary.
+
+  Background:
+    Given I am on the YourDetailsPage
+
+    Scenario Outline:
+      Given I enter my age as "<age>"
```

Comment:
I agree steps should be small, but I don't think so much logic should be in the feature itself. There's a really good section in The Cucumber book on my desk.. afraid I can't tell you where from here though! 🙈